CMakeLists.txt

```diff
@@ -66,6 +66,9 @@ add_subdirectory(Dependencies/SafeDisc)
 add_subdirectory(Dependencies/Utility)
 add_subdirectory(resources)
 
+# Add the common build targets
+add_subdirectory(Common)
+
 # Add main build targets
 if(GENZH_BUILD_ZEROHOUR)
     add_subdirectory(GeneralsMD)
```

Comment:
When this progresses more and more, what kind of left-overs do we expect inside `Generals` and `GeneralsMD`, roughly?